docs/units/equivalencies.rst

```diff
@@ -463,15 +463,20 @@ point of 3631.1 Jy::
 Temperature Equivalency
 -----------------------
 
-The :func:`~astropy.units.temperature` equivalency allows conversion
-between the Celsius, Fahrenheit, Rankine and Kelvin.
+Conversions between kelvins and degrees Rankine do not require using any
+equivalencies because the Kelvin scale and the Rankine scale have the same zero
+point::
 
-Example
-^^^^^^^
-
-.. EXAMPLE START: Using the Temperature Equivalency
+   >>> temp_K = 100 * u.K
+   >>> temp_R = temp_K.to(u.imperial.deg_R)
+   >>> temp_R
+   <Quantity 180. deg_R>
+   >>> temp_R.to(u.K)
+   <Quantity 100. K>
 
-To convert between temperature scales::
+The Celsius and Fahrenheit scales, however, have different zero points, which
```

Comment:
The example was the use of the equivalency, so in revising this, it should it should start just above this line. To make it self-contained, delete ", however,"